jobs/release.Jenkinsfile

```diff
@@ -259,26 +259,28 @@ lock(resource: "release-${params.STREAM}", extra: locks) {
         if (push_containers) {
             stage("Push Containers") {
                 parallel push_containers.collectEntries{configname, val -> [configname, {
-                    if (!registry_repos?."${configname}"?.'repo') {
+                    if (!registry_repos?."${configname}") {
                         echo "No registry repo config for ${configname}. Skipping"
                         return
                     }
                     withCredentials([file(variable: 'REGISTRY_SECRET',
                                           credentialsId: 'oscontainer-push-registry-secret')]) {
-                        def repo = registry_repos[configname]['repo']
+                        def repos = registry_repos[configname]
                         def (artifact, metajsonname) = val
-                        def tag_args = registry_repos[configname].tags.collect{"--tag=$it"}
-                        def v2s2_arg = registry_repos.v2s2 ? "--v2s2" : ""
-                        shwrap("""
-                        export COSA_SUPERMIN_MEMORY=1024 # this really shouldn't require much RAM
-                        cp \${REGISTRY_SECRET} tmp/push-secret-${metajsonname}
-                        cosa supermin-run /usr/lib/coreos-assembler/cmd-push-container-manifest \
-                            --auth=tmp/push-secret-${metajsonname} \
-                            --repo=${repo} ${tag_args.join(' ')} \
-                            --artifact=${artifact} --metajsonname=${metajsonname} \
-                            --build=${params.VERSION} ${v2s2_arg}
-                        rm tmp/push-secret-${metajsonname}
-                        """)
+                        for (repo in repos) {
+                            def tag_args = repo.tags.collect{"--tag=$it"}
+                            def v2s2_arg = repo.v2s2 ? "--v2s2" : ""
+                            shwrap("""
+                            export COSA_SUPERMIN_MEMORY=1024 # this really shouldn't require much RAM
+                            cp \${REGISTRY_SECRET} tmp/push-secret-${metajsonname}
+                            cosa supermin-run /usr/lib/coreos-assembler/cmd-push-container-manifest \
```

Comment:
Reviewing this with JB, one thing we remembered is that registry.ci doesn't actually support manifests. So we need to fall back to `cosa copy-container` instead here which has the magic "push manifested images as separate images with an arch suffix tag". (Or, add that logic to `cmd-push-container-manifest` which feels a bit weird given its name...)

Reply:
Made the change [c206a72ed604e10d364f981e29bc99f927615fc3](https://github.com/coreos/fedora-coreos-pipeline/pull/1091/commits/c206a72ed604e10d364f981e29bc99f927615fc3)
This will publish an extra tag on the destination registry as right now `copy-container` does not support differentiating source tag and destination tags. I think that's OK though as is the majority of case that's going to be `$VERSION` and will end up being the same.

https://github.com/coreos/coreos-assembler/blob/1f84d64cd85aa2401bdcf398d4210ea4954a0790/src/cmd-copy-container#L117

Reply:
cool !

Reply:
Hopefully I had made a separate commit